src/main/kotlin/com/terraformation/backend/tracking/db/PlantingSiteStore.kt

```diff
@@ -42,40 +43,43 @@ class PlantingSiteStore(
 
   private val monitoringPlotBoundaryField = MONITORING_PLOTS.BOUNDARY.forMultiset()
   private val plantingSubzoneBoundaryField = PLANTING_SUBZONES.BOUNDARY.forMultiset()
-  private val plantingSitesBoundaryField = PLANTING_SITES.BOUNDARY
   private val plantingZonesBoundaryField = PLANTING_ZONES.BOUNDARY.forMultiset()
 
   fun fetchSiteById(
       plantingSiteId: PlantingSiteId,
-      includeSubzones: Boolean = false,
-      includePlots: Boolean = false,
+      depth: PlantingSiteDepth,
   ): PlantingSiteModel {
     requirePermissions { readPlantingSite(plantingSiteId) }
 
-    val zonesField = plantingZonesMultiset(includeSubzones, includePlots)
+    val zonesField = if (depth != PlantingSiteDepth.Site) plantingZonesMultiset(depth) else null
 
     return dslContext
-        .select(PLANTING_SITES.asterisk(), plantingSitesBoundaryField, zonesField)
+        .select(PLANTING_SITES.asterisk(), zonesField)
         .from(PLANTING_SITES)
         .where(PLANTING_SITES.ID.eq(plantingSiteId))
-        .fetchOne { record -> PlantingSiteModel(record, plantingSitesBoundaryField, zonesField) }
+        .fetchOne { record -> PlantingSiteModel(record, zonesField) }
         ?: throw PlantingSiteNotFoundException(plantingSiteId)
   }
 
   fun fetchSitesByOrganizationId(
       organizationId: OrganizationId,
-      includeZones: Boolean = false,
+      depth: PlantingSiteDepth = PlantingSiteDepth.Site,
   ): List<PlantingSiteModel> {
     requirePermissions { readOrganization(organizationId) }
 
-    val zonesField = if (includeZones) plantingZonesMultiset(true, false) else null
+    val zonesField =
+        if (depth != PlantingSiteDepth.Site) {
+          plantingZonesMultiset(depth)
+        } else {
+          null
+        }
 
     return dslContext
-        .select(PLANTING_SITES.asterisk(), plantingSitesBoundaryField, zonesField)
+        .select(PLANTING_SITES.asterisk(), zonesField)
```

Comment:
just curious , is the sites boundary field already covered by the asterisk?

Reply:
Yes, it is. This extra argument is a remnant from when we were doing on-the-fly coordinate system transfers at read time. It should really have been removed in commit 6a94d3c9 but I missed it then.